bindata/network/ovn-kubernetes/006-ovnkube-config.yaml

```diff
@@ -0,0 +1,34 @@
+---
+# The config file used by ovnkube is passed as a configmap.
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: ovnkube-config
+  namespace: openshift-ovn-kubernetes
+data:
+  config:   |-
+    [default]
+    mtu="{{.MTU}}"
+
+    [kubernetes]
+    service-cidr="{{.OVN_service_cidr}}"
+    cluster-subnets={{.K8S_cluster_subnets}}
+    ovn-config-namespace="{{.OVNKUBE_namespace}}"
```

Comment:
@pecameron for OpenShift this value will always be "openshift-ovn-kubernetes" so I think we can just hardcode it here for the CNO. Upstream of course it needs to be configurable but will default to "ovn-kubernetes".